code/index.html

```diff
@@ -5,17 +5,54 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project Name</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" type="text/css" href="style.css"/>
     <!-- If you're using google fonts, that should be added here -->
   </head>
   <body>
-    <!-- 
-      Write your code here inside the body to make it appear in the browser. 
-      Below is an example on how to use an image tag. Copy it and add it in the body, but outside the comment tag if you want to see how it looks like on the website. Then you can change the size and other styling for it in the css file.
+    <div>
+      <menu id="anker">
+       <a href="./index.html">Home</a>
+       <a href="./about.html">Society</a>
+       <a href="./photos.html">Travel</a>
+       <a href="./photos.html">Leadership</a>
```

Comment:
If you want to create "dummy"-links (for when there are no real pages to link to, you can write `href="#"`. That way you will not get an error when you click the links.